readthedocs/search/parsers.py

```diff
@@ -153,9 +153,40 @@ def _parse_sections(self, title, body):
                         'title': title,
                         'content': content,
                     }
+                except Exception as e:
+                    log.info("Unable to index section.", section=str(e))
+
+        dls = body.css("dl")
+        for dl in dls:
+            dts = dl.css("dt")
+
+            for dt in dts:
+                try:
+                    title, id = self._parse_dt(tag)
+                    next_element = dt.next
+                    # We only index a dt with an accompanying dd
+                    if next_element.tag != "dd":
+                        continue
+                    content, _ = self._parse_section_content(next_element, depth=2)
```

Comment:
I think we want to index everything that's inside, not till we find another section, but we may be able to re-use this function.

Reply:
yeah, but I mean that we need to index everything that's inside the `dd` element, currently `_parse_section_content` will index things inside the dd element till it finds a section inside it.

Reply:
Isolated example is now:

```
from selectolax.parser import HTMLParser
dls=HTMLParser("<dl><dt>term a</dt><dd>description a</dd><dt>term b</dt><dt>term c</dt><dd>description b</dd></dl>").css("dl")

for dl in dls:
    dts = dl.css("dt")

    for dt in dts:
        print(dt.text())
        next_element = dt.css_first("dt ~ dd")
        # We only index a dt with an accompanying dd
        if not next_element or next_element.tag != "dd":
            continue
        print(next_element.text())
```